doc/win32-usb-troubleshoot.md

```diff
@@ -0,0 +1,31 @@
+# Troubleshooting LIBUSB_* errors on win32 platforms
+
+nRF Connect Desktop uses [libusb](https://libusb.info/) and [node-usb](https://github.com/tessel/node-usb) to know how many USB-capable Nordic SoCs (e.g. nRF52840) are connected to your PC, and also to set them into DFU mode when they need to be re-programmed by a nRF Connect Desktop application.
```

Comment:
s/a nRF/an nRF/

Reply:
Not really. "an" is used when the *pronunciation* of an abbreviation starts with a vowel. See https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms-initialisms

Reply:
Exactly, and since you pronounce "en-er-ef" you have to write 'an nRF'.

Reply:
Ha, I thought I wrote it the other way. Oh well.

Reply:
Yes, it is "an" here.